webapp/src/components/AssetPage/BestBuyingOption/BestBuyingOption.tsx

```diff
@@ -0,0 +1,252 @@
+                <span className={styles.informationBold}>
+                  <Mana
+                    withTooltip
+                    size="medium"
+                    network={listing.order.network}
+                  >
+                    {formatWeiMANA(listing.order.price)}
+                  </Mana>
+                </span>
+
+                <div className={styles.informationText}>
+                  <ManaToFiat
+                    mana={ethers.utils
+                      .parseEther(formatWeiMANA(listing.order.price))
+                      .toString()}
+                  />
+                </div>
+              </div>
+            </div>
+            {bid && (
+              <div className={styles.containerColumn}>
+                <span className={styles.informationTitle}>
+                  {t(
+                    'best_buying_option.buy_listing.highest_offer'
+                  ).toUpperCase()}
+                </span>
+                <div className={styles.containerRow}>
+                  <div className={styles.informationBold}>
+                    <Mana
+                      withTooltip
+                      size="medium"
+                      network={listing.order.network}
+                    >
+                      {formatWeiMANA(bid.price)}
+                    </Mana>
+                  </div>
+
+                  <div className={styles.informationText}>
+                    <ManaToFiat
+                      mana={ethers.utils
+                        .parseEther(formatWeiMANA(bid.price))
+                        .toString()}
+                    />
+                  </div>
+                </div>
+              </div>
+            )}
+            <div className={styles.containerColumn}>
+              <span className={styles.informationTitle}>
+                {t('best_buying_option.buy_listing.issue_number').toUpperCase()}
+              </span>
+              <span className={styles.informationText}>
+                <span className={styles.informationBold}>
+                  {listing.order.tokenId}
+                </span>
+                /{listing.total}
+              </span>
+            </div>
+            <div className={styles.containerColumn}>
+              <span className={styles.informationTitle}>
+                {t('best_buying_option.buy_listing.owner').toUpperCase()}
+              </span>
+              <LinkedProfile
+                className={styles.linkedProfileRow}
+                address={listing.order.owner}
+              />
+            </div>
+          </div>
+          <div className={styles.containerRow}>
+            <BuyNFTButtons asset={asset} />
+          </div>
+          <Button
+            href={locations.nft(asset.contractAddress, listing.order.tokenId)}
+          >
+            {t('best_buying_option.buy_listing.view_listing')}
+          </Button>
+          <span>
+            {t('best_buying_option.buy_listing.expires')}
```

Comment:
```suggestion
            {t('best_buying_option.buy_listing.expires')}{' '}
```
to fix this:
![image](https://user-images.githubusercontent.com/8763687/226612281-220b1da2-a759-4266-924a-bf6d9856200d.png)